edge/pkg/edged/apis/types.go

```diff
@@ -2,7 +2,10 @@ package apis
 
 import (
 	"errors"
+	"github.com/docker/docker/api/types/container"
```

Comment:
place this import below the "time" import along with the kubernetes import.

Reply:
done